Dockerfile

```diff
@@ -1,15 +1,12 @@
-# Alpine is a lightweight Linux
-FROM mhart/alpine-node:6
+FROM node:6-alpine
 
-# Update latest available packages
-RUN apk update && \
-    apk add ffmpeg && \
-    apk add git && \
```

Comment:
I seem to recall that git was required by bower. Does it come by default in the official image now?

Reply:
It's still there, I just put it on the same line as ffmpeg

Reply:
Got it. Thanks.